components/calendar/calendar.js

```diff
@@ -449,13 +464,15 @@ class Calendar extends LocalizeStaticMixin(LitElement) {
 				} else {
 					numDaysChange = -1;
 				}
+				preventDefault = true;
```

Comment:
Stops scrolling?

Reply:
Stops safari voiceover from behaving not expectedly - it seems to go deep into the text when left/right were pressed rather than staying at the cell level, which was causing it not to read the aria-label.

Reply:
Cool -- can you throw in a comment saying something to that effect? That way we know what it's for in the future and better judge if we can clean it up.